packages/@uppy/webcam/src/index.js

```diff
@@ -72,6 +76,8 @@ module.exports = class Webcam extends UIPlugin {
       </svg>
     )
 
+    this.#enableMirror = opts?.mirror ?? true
```

Comment:
Instead you can put this line somewhere after line 117, it would simplify greatly the assignment and would avoid duplicating the default value (it's already defined line 109):
```suggestion
    this.#enableMirror = this.opts.mirror
```